From: Jonathan Kamens <jik@kamens.us>
Date: Sun, 18 Apr 2021 09:51:50 -0400
Subject: Fix typo in log message ("ststus" => "status")

---
 runjob.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/runjob.c b/runjob.c
index b5ed26c..f51b2e6 100644
--- a/runjob.c
+++ b/runjob.c
@@ -195,7 +195,7 @@ tend_mailer(job_rec *jr, int status)
 {
     if (WIFEXITED(status) && WEXITSTATUS(status) != 0)
 	complain("Tried to mail output of job `%s', "
-		 "but mailer process (" SENDMAIL ") exited with ststus %d",
+		 "but mailer process (" SENDMAIL ") exited with status %d",
 		 jr->ident, WEXITSTATUS(status));
     else if (!WIFEXITED(status) && WIFSIGNALED(status))
 	complain("Tried to mail output of job `%s', "
